app/helpers/check_answers_helper.rb

```diff
@@ -12,12 +12,25 @@ def display_answered_questions_summary(subsection, case_log)
   end
 
   def get_answer_label(question, case_log)
-    answer = question.prefix == "£" ? ActionController::Base.helpers.number_to_currency(question.answer_label(case_log), delimiter: ",", format: "%n") : question.answer_label(case_log)
-
-    if answer.present?
-      [question.prefix, answer, question.suffix].join("")
+    if question.id == "earnings"
```

Comment:
Is there a more generic way of doing this? I think if we start hard coding special cases like this in, then pretty soon the whole JSON definition idea becomes a bit useless and we might as well make every page a bespoke view?